yarn.lock

```diff
@@ -6048,30 +6048,10 @@ caniuse-api@^3.0.0:
     lodash.memoize "^4.1.2"
     lodash.uniq "^4.5.0"
 
-caniuse-lite@^1.0.0, caniuse-lite@^1.0.30001449:
```

Comment:
Please do not downgrade caniuse-lite.

Reply:
This was a previous commit, it was resolved earlier